aperturedb/Connector.py

```diff
@@ -280,7 +281,7 @@ def query(self, q, blobs=[]):
             self.response, self.blobs = self._query(q, blobs)
             self.last_query_time = time.time() - start
             return self.response, self.blobs
-        except:
+        except BaseException:
```

Comment:
I was thinking if this could be also propagated back to the user, something like : 
except BaseException as e:
   raise ConnectionError ("ApertureDB disconnected" + e.msg)

Reply:
makes sense, I like adding context to traces. this was added by autopep though.

Would you like me to go and add the suggested code you have here?

Reply:
Oh I see. We can create it under a new Issue then. I thought you were made the changes.
I ran into a situation with the way we print the message here, without giving back the entire information.